### ISSUE TITLE
🐛Bug: TypeScript requires to override all the id's when using `ids` prop for the builder of `Combobox`

### ISSUE THREAD

**Issue by oscard0m** [melt-ui/melt-ui#780]
**Describe the bug**
TypeScript requires to override all the id's when using `ids` prop for the builder of `Combobox` even if you only want to override one of the id's:

![image](https://github.com/melt-ui/melt-ui/assets/2574275/9aaa0625-0258-4873-9b30-fc0126d947f6)

**To Reproduce**
Using `melt-ui` `>=v0.58`:

```js
<script lang="ts" generics="T extends Option">
import { createCombobox } from '@melt-ui/svelte';

const { ... } = createCombobox<T>({
    ...
    ids: {
       trigger: name,
    },
});
</script>
```
**Expected behavior**
To have the option to override a certain `id`, and not all of them without TS complaining.

**Additional context**
- Discord thread: https://discord.com/channels/1116682155809067049/1178970315276177460/1178975784665362453 (it includes examples with Stackblitz)

**Comment by TGlide** [comment]
This is already being addressed :) #757

**Comment by abdel-17** [comment]
Hey seems this issue has been fixed. Do you mind closing it?
